test/integration/scheduler/util.go

```diff
@@ -105,6 +108,10 @@ func InitRegistryAndConfig(t *testing.T, factory func(plugin framework.Plugin) f
 		plugin := configv1.Plugin{Name: p.Name()}
 
 		switch p.(type) {
+		case framework.QueueSortPlugin:
+			pls.QueueSort.Enabled = append(pls.QueueSort.Enabled, plugin)
+			// It's intentional to disable the PrioritySort plugin.
```

Comment:
BTW, why are we using a new fake plugin instead of testing `PrioritySort` directly?